src/RazorSdk/SourceGenerators/Microsoft.NET.Sdk.Razor.SourceGenerators.csproj

```diff
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="$(MicrosoftCodeAnalysisCSharpPackageVersion)"/>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.10.0-3.21224.14"/>
```

Comment:
Out of curiosity, what necessitates this specific version? Could we also add a comment to say why we don't use the value from Versions.props?

Reply:
I wanted to make sure we pick up the recent MSBuild config parsing changes. I'll add a comment as to why.